basic/ani.txt

```diff
@@ -22,10 +23,8 @@ M%=M%+1
 NEXT
 D%=?M%
 M%=M%+1
-K%=INKEY(D%-2)
-NEXT
-M%=N%
-UNTIL 0
+K%=INKEY(D%-2):IFK%=-1NEXT:M%=N%:UNTIL0
+CLS:RUN
```

Comment:
Clear screen so you don't see residual old image data when prog next switches back to MODE2.